core/src/epicli/data/common/defaults/configuration/postgresql.yml

```diff
@@ -1,7 +1,38 @@
 kind: configuration/postgresql
-title: "Postgresql"
+title: PostgreSQL
 name: default
 specification:
-  version: '10'
   replication:
-    enable: no
\ No newline at end of file
+    enable: no
+  config_file:
+    parameter_groups:
+      - name: CONNECTIONS AND AUTHENTICATION
+        subgroups:
+          - name: Connection Settings
+            parameters:
+              - name: listen_addresses
+                value: "'*'"
+                comment: listen on all addresses
+      - name: ERROR REPORTING AND LOGGING
+        subgroups:
+          - name: Where to Log
+            parameters:
+              - name: log_filename
+                value: "'postgresql-%Y-%m-%d_%H%M%S.log'"
+  extensions:
+    pgaudit:
+      enabled: true
+      config_file_parameters:
+        shared_preload_libraries: SET_BY_AUTOMATION
```

Comment:
This also should be defined on per module basis and not set by automation.

Reply:
Done